### ISSUE TITLE
Unable to build

### ISSUE THREAD

**Issue by aminemarref** [h5p/h5p-mark-the-words#103]
Running `npm install` complains about missing `package.json`.

**Comment by otacke** [comment]
@aminemarref There's nothing to build, hence no package.json file.

**Comment by 100jiaxing** [comment]
Could you tell me how to use it? I tried to upload to the upload location of h5p.org, but it requires a file with the .h5p suffix.

**Comment by otacke** [comment]
@100jiaxing h5p.org is a site for testing H5P on a live system, you cannot upload library files there. And you normally also do not require to build libraries on your own. You can install H5P on your own platform (see https://h5p.org/installation) and install content types from within the H5P Hub.

**Comment by 100jiaxing** [comment]
Yes, thank you for your reply. I use wordpress and I have the h5p plugin installed, but it offers limited content types. For example, I want to use the https://github.com/sherin-shifana/h5p-mark-the-letters.git card feature, but I don't know how to use it

**Comment by 100jiaxing** [comment]
> Yes, thank you for your reply. I use wordpress and I have the h5p plugin installed, but it offers limited content types. For example, I want to use the https://github.com/sherin-shifana/h5p-mark-the-letters.git card feature, but I don't know how to use it

Yes, thank you for your reply. I use wordpress and I have the h5p plugin installed, but it offers limited content types. For example, I want to use the https://github.com/sherin-shifana/h5p-mark-the-letters.git card feature, but I don't know how to use it

**Comment by otacke** [comment]
@100jiaxing You could consult https://peter.baumgartner.name/2021/04/12/h5p-library-installation/ for instance or ask on the H5P forums. An issue of some completely unrelated repository is hardly the right place to get support.

**Comment by 100jiaxing** [comment]
ok i see ，thinks

**Comment by otacke** [comment]
@aminemarref I assume you could close this issue, right?
